.gitignore

```diff
@@ -30,8 +30,5 @@ $RECYCLE.BIN/
 node_modules/
 bower_components/
 .tmp
-dist
```

Comment:
is there a reason these needed to be removed?

Reply:
Same as comment on top about package.json.